modules/tinymce/src/core/main/ts/api/UndoManager.ts

```diff
@@ -50,6 +51,27 @@ const UndoManager = (editor: Editor): UndoManager => {
       return Rtc.addUndoLevel(editor, undoManager, index, locks, beforeBookmark, level, event);
     },
 
+    /**
+     * Dispatch a change event if the content in last layer of the undoManager and the current editor content are different
+     *
+     * @method dispatchIfChanged
+     */
+    dispatchIfChanged: (): void => {
```

Comment:
We need to remove the `EditorUpload` logic and make it use this new API as well.

- [ ] Replace old editor upload logic

Reply:
ok, I opened a thread

Reply:
ok, I simplified the API and moved the check in `EditorUpload`